### PR TITLE
AB#705 Refactor quickstart

### PR DESCRIPTION
**Pull request by m1ghtym0**

### CODE REVIEW

**Comment by thomasten** [comment]
Merge into main instead? IMO we should use dev only for docu of unreleased features.

File: content/docs/getting-started/quickstart.md [hunk 1]
```diff
@@ -7,182 +7,146 @@ weight: 1
...
+The first step is to install Marblerun's *Coordinator* control-plane. To do this, run:
 
 ```bash
-helm install -f ./kubernetes/nosgx_values.yaml emojivoto ./kubernetes --create-namespace -n emojivoto
+marblerun install --simulation
 ```
 
-## Step 6: Watch it run
+In this command, the marblerun install command generates a Kubernetes manifest with all the necessary control plane resources.
+The simulation flag tells Marblerun that real SGX hardware might not be present and the SGX-layer should be emulated.
```
Comment (katexochen) [inline content/docs/getting-started/quickstart.md:66]: Maybe we should already mention in the intro that our demo does not use SGX and include a warning that this is not for production ;)

File: content/docs/getting-started/quickstart.md [hunk 2]
```diff
@@ -7,182 +7,146 @@ weight: 1
...
 
-## Step 3: Initialize and verify the Coordinator
+## Step 5: Watch it run
 
-Get the Coordinator's address and set the DNS. Check our docs on [how to expose the Client-API]({{< ref "docs/tasks/deploy.md#accesing-the-client-api" >}})
+You can now check the Marblerun log and see the services being authenticated by the coordinator.
 
 ```bash
-export MARBLERUN=mycluster.uksouth.cloudapp.azure.com
+kubectl -n marblerun logs -f -ledgeless.systems/control-plane-component=coordinator
```
Comment (katexochen) [inline content/docs/getting-started/quickstart.md:129]:
```suggestion
kubectl -n marblerun logs -f -edgeless.systems/control-plane-component=coordinator
```
Not sure if this is a typo....

**Review by katexochen — CHANGES_REQUESTED** [review]
Nice Quickstart! feels very welcoming :smile_cat:

File: content/docs/getting-started/quickstart.md (hunk 2 above)
Comment (m1ghtym0) [reply content/docs/getting-started/quickstart.md:129]: No, that's actually correct. `-l` means filtering by label

**Comment by m1ghtym0** [comment]
I used the "we" form intentionally to have an "engaging" touch in the quickstart.
In the sense of "we're in this together, let's work through it".
Do you think that makes it sound less professional?

**Comment by katexochen** [comment]
I think it is nice to use "we" in a context of "we are going to this step by step together". However, in my opinion it should only be used if this is indeed the case. If the user needs to install prerequisites before and without our help, it should state "you have to install ... first". The cluster used in a tutorial is the users cluster, not ours. It is nice to use "we" on a higher level view like "the next steps we are going to face are...", but atomar instuctions should still be in imparative, "run command foo ", "install bar with ...".

**Review by katexochen — APPROVED** [review]

**Pull request merged**